plugins/chat/assets/javascripts/discourse/components/chat-channel-unread-indicator.js

```diff
@@ -1,46 +1,3 @@
-import discourseComputed from "discourse-common/utils/decorators";
```

Comment:
You can just delete this file. Lone `.hbs` files in the `discourse/components/` directory are automatically treated as template-only glimmer components